### PR TITLE
[OSDOCS-11718]: Edit ingress docs incorrectly identify "non-default" ingresses

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.17+
Issue:

https://issues.redhat.com/browse/OSDOCS-11718

https://80588--ocpdocs-pr.netlify.app/openshift-rosa/latest/cli_reference/rosa_cli/rosa-manage-objects-cli.html#rosa-edit-ingress_rosa-managing-objects-cli
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by wgordon17** [comment]
LGTM, thank you!

**Comment by mletalie** [comment]
Hello 
@xueli181114 , 
Could I please get a QE review for this PR when you get a moment? Thx!

**Review by lahinson — COMMENTED** [review]
LGTM

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed
/label merge-review-needed

File: modules/rosa-edit-objects.adoc [hunk 1]
```diff
@@ -79,8 +79,12 @@ $ rosa edit cluster --cluster=mycluster --interactive
 [id="rosa-edit-ingress_{context}"]
 == edit ingress
 
-Edits the additional non-default application router for a cluster.
+Edits the default application router for a cluster.
 
+[NOTE]
+====
+For information about editing non-default application routers, see _Additional resources_.
```
Comment (michaelryanpeter) [inline modules/rosa-edit-objects.adoc:86]: Nit:
```suggestion
For information about editing non-default application routers, see "Configuring the Ingress Controller".
```

Re: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

**Review by michaelryanpeter — COMMENTED** [review]
This is a a very minor style issue, and I don't think it is worth blocking a merge on. Would you please fix this up in a follow up PR?

Thanks!

**Comment by mletalie** [comment]
> This is a a very minor style issue, and I don't think it is worth blocking a merge on. Would you please fix this up in a follow up PR?
> 
> Thanks!
Yes, will make a note to update in future PR. Thanks!

**Pull request merged**